netmap/json_test.go

```diff
@@ -80,3 +80,25 @@ func TestPlacementPolicy_Interopability(t *testing.T) {
 		})
 	}
 }
+func BenchmarkManySelects(b *testing.B) {
+	testsFile := filepath.Join("json_tests", "many_selects.json")
+	bs, err := ioutil.ReadFile(testsFile)
+	require.NoError(b, err)
+
+	var tc TestCase
+	require.NoError(b, json.Unmarshal(bs, &tc))
+	tt, ok := tc.Tests["Select"]
+	require.True(b, ok)
+
+	b.ResetTimer()
+	b.ReportAllocs()
+
+	for i := 0; i < b.N; i++ {
+		nodes := NodesFromInfo(tc.Nodes)
+		nm, err := NewNetmap(nodes)
+		require.NoError(b, err)
+
+		_, err = nm.GetContainerNodes(&tt.Policy, tt.Pivot)
+		require.NoError(b, err)
```

Comment:
What function do we bench here? Depending on this we might `StopTimer` for preparation routines. Also, lets avoid `require` in benchmarks, as it can affect the results. `if err != nil { b.FailNow() }`.

Reply:
I think we can keep only `GetContainerNodes` function.